@here/harp-mapview/lib/MapView.ts

```diff
@@ -2030,6 +2051,13 @@ export class MapView extends THREE.EventDispatcher {
         this.m_elevationProvider = elevationProvider;
     }
 
+    /**
+     * Draw a new frame synchronously.
+     */
+    redraw() {
```

Comment:
`renderSync` ?

If this is public, then maybe remove this bool attribute from `update`.
Update is meant mainly for other APIs to inform MAP, that update is needed, not to really re-render.

You should also add `if (!this.m_updatePending) { return}` to `private render()` so, if we rendered stuff asynchronously, nobody will try to redraw it again. Or similar mechanism.

Reply:
There is already a method renderFunc where you can pass in the time, is that not good enough?

Reply:
Following other comments, like 

>  there is some code to cancel an active requestAnimationFrame, is this necessary here also?

So, looks like `renderFunc` is intended to be called by `rAF`, so it assumes that frame was canceled - no need to cancel.

This, one on the other hand can cancel requested rAF as we know it won't be needed (unless new update is scheduled), so proposed

> `if (!this.m_updatePending) { return} to private render()`

is not needed.